orchestra/models/core/models.py

```diff
@@ -659,6 +660,31 @@ class Meta:
     activity_log = JSONField(default={'actions': []})
 
 
+class TodoQA(TodoQAMixin, BaseModel):
```

Comment:
Do we care who the reviewer is?

Reply:
I don't think so. At this point the reviewer will be the admin user.